bin/cron/ops_data_pull

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env ruby
+
+require_relative '../../dashboard/config/environment'
+require 'cdo/properties'
+include ApplicationHelper
+include UsersHelper
+
+pd_scripts = Script.all.select { |script| !!script.pd }
```

Comment:
I guess there isn't any reason, i thought i might have needed to convert 'false' strings but I guess `!!` doesn't even handle that case. i didn't know we added `?` methods to serialized properties now, that's helpful!